recipe/meta.yaml

```diff
@@ -1,31 +1,32 @@
 {% set name = "azure-storage-file-share" %}
-{% set version = "12.10.1" %}
+{% set version = "12.15.0" %}
 
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
-  sha256: 5679bbd9c760ffd3fe27749c9e3e63706ea42c274854c1a7c6753d6a7da8c464
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0098f1cfa9344c4f781ce0cd504ff3a35255751553c01e720dccd0caa8d8aa2a
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   host:
     - pip
-    - python >=3.7
+    - python
+    - wheel
+    - setuptools
   run:
-    - python >=3.7
+    - python
     - azure-core >=1.26.0,<2.0.0
     - cryptography >=2.1.4
-    - typing_extensions >=4.0.1
+    - typing-extensions >=4.3.0
     - isodate >=0.6.1
-    - msrest >=0.7.1
```

Comment:
they have a sort of recipe there, and `msrest` seems to be included:
https://github.com/Azure/azure-sdk-for-python/blob/azure-storage-file-share_12.15.0/conda/conda-recipes/azure-storage/meta.yaml

even though, I see from the `setup.py` that it is not included:
https://github.com/Azure/azure-sdk-for-python/blob/azure-storage-file-share_12.15.0/sdk/storage/azure-storage-file-share/setup.py

Reply:
The source of truth to me is PyPI and I see no imports of msrest:
```
$ grep msrest -r .
./PKG-INFO:- Removed `msrest` dependency.
./PKG-INFO:- Fixed msrest dependency issue (#16250)
./CHANGELOG.md:- Removed `msrest` dependency.
./CHANGELOG.md:- Fixed msrest dependency issue (#16250)
./azure/storage/fileshare/_generated/_serialization.py:        msrest type and 'key' with the RestAPI encoded key.
./azure/storage/fileshare/_generated/_serialization.py:            # https://github.com/Azure/msrest-for-python/issues/197
./azure/storage/fileshare/_generated/_serialization.py:            # https://github.com/Azure/msrest-for-python/issues/197
./azure_storage_file_share.egg-info/PKG-INFO:- Removed `msrest` dependency.
./azure_storage_file_share.egg-info/PKG-INFO:- Fixed msrest dependency issue (#16250)
```